configurations/forwardNew.yml

```diff
@@ -0,0 +1,18 @@
+features:
+  beampipe:
+  tracking:
+    vertex_barrel:
+    silicon_barrel:
+    mpgd_barrel:
+    silicon_disks:
+    support_service_assembly:
+    tof_barrel:
+    tof_endcap:
+  pid:
+    dirc:
+    drich:
+  ecal:
+    forward_homogeneous:
+    forward_insert_homogeneous:
+  hcal:
+    lfhcal:
```

Comment:
I don't think forwardNew is a clear name (or even correct as this includes also barrel detectors). Can we rename this to `full_hcal_only_forward` or so?